nncf/hw_configs/vpu.json

```diff
@@ -40,28 +47,28 @@
             },
             "quantization": {
                 "activations": ["q8_tn", "q4_tn"],
-                "weights": ["q8_ch","q4_ch", "q2_ch"]
+                "weights": ["q8_w", "q4_w", "q2_w"]
```

Comment:
What are `q2_w and q4_w`? they are not defined.

Reply:
Thanks for spotting this, fixed.